.travis.yml

```diff
@@ -39,6 +39,17 @@ matrix:
       env:
         - JOBCACHE=3
 
+    - name: Lint code with clippy
```

Comment:
Does it make sense to combine the rustfmt check with this check to reduce the amount of different jobs? We could add `cargo clippy --all` to the rustfmt job above.

Reply:
Is there a way to make them appear in a different rows in the Travis output? I'd like to spare user from looking inside the Travis row to see what's going on. He sees that formatter failed, goes to his own terminal, runs it, creates new commit.

Reply:
Okay, that sounds fair to me.